src/main.ts

```diff
@@ -22,24 +22,38 @@ export async function main(): Promise<Outputs> {
     const deploymentName = getInput('deploymentName')
     const parameters = getInput('parameters')
     const managementGroupId = getInput('managementGroupId')
+    const failOnStdErr = getInput('failOnStdErr')
```

Comment:
you should be able to use `core.getBooleanInput()` function: https://github.com/actions/toolkit/tree/main/packages/core#inputsoutputs

This will remove need to do conversion later in L33

Reply:
What I am not sure is if it will take automatically default value specified in action.yml if user does not specify it?

Reply:
If not so, then you can `|| true`

Reply:
@bishal-pdMSFT This method is not available in the current version of `@actions/core` which we are using. Should we update the package then?